README.md

```diff
@@ -47,11 +48,21 @@ Then you can visit:
 config-ui by url `http://YOUR-NODE-IP:30090`
 grafana by url `http://YOUR-NODE-IP:30091`
 
-## Update
+## Upgrade
+
+Note:
+If you're upgrading from DevLake v0.17.x or earlier versions to v0.18.x or later versions:
+
+1. Please copy the ENCODE_KEY value from /app/config/.env of the lake pod (eg: devlake-lake-0), and replace <ENCRYPTION_SECRET> of below upgrade cmd.
```

Comment:
Can you update to:
1. Copy the ENCODE_KEY value from `/app/config/.env` of the lake pod (e.g. devlake-lake-0), and replace the <ENCRYPTION_SECRET>  in the upgrade command below.